marklogic-client-api/src/test/java/com/marklogic/client/test/dataservices/CallBatcherDefaultArgsTest.java

```diff
@@ -98,8 +98,8 @@ public void singleAtomicCallFieldTest() {
                     fail("atomic batcher failed");
                     });
 
-        batcher.startJob​();
-        batcher.add(caller.args().param("param1", 1.1));
+        batcher.getDataMovementManager().startJob(batcher);
```

Comment:
The start() in the previous version of line 101 is supported (and probably better than the revision because simpler).

Also, just so you know, adding arguments implicitly starts a job.